curriculum/challenges/english/25-front-end-development/workshop-library-manager/67116d7584d0b469b14579bf.md

```diff
@@ -0,0 +1,91 @@
+---
+id: 67116d7584d0b469b14579bf
+title: Step 2
+challengeType: 1
+dashedName: step-2
+---
+
+# --description--
+
+Inside the `library` array, create an object with the following properties and values:
+
+| Property | Value   |
+| ----------- | ------- |
+| `title` | `"Your Next Five Moves: Master the Art of Business Strategy"`|
+| `author` | `"Patrick Bet-David and Greg Dinkin"`|
+| `about` | `"A book on how to plan ahead"`|
+| `pages` | `320` |
+
+# --hints--
+
+You should have an object inside the `library` array.
+
+```js
+assert.isObject(library[0])
+```
+
+Your object should have a `title` property.
+
+```js
+assert.property(library[0], "title")
+```
+
+The value of your `title` property should be `"Your Next Five Moves: Master the Art of Business Strategy"`.
+
+```js
+assert.equal(library[0]?.title, "Your Next Five Moves: Master the Art of Business Strategy");
+```
+
+Your object should have an `author` property.
+
+```js
+assert.property(library[0], "author")
+```
+
+Your `author` property should have the value `"Patrick Bet-David and Greg Dinkin"`.
+
+```js
+assert.equal(library[0]?.author, "Patrick Bet-David and Greg Dinkin");
+```
+
+Your object should have an `about` property.
+
+```js
+assert.property(library[0], "about")
+```
+
+Your `about` property should have the value `"A book on how to plan ahead"`.
+
+```js
+assert.equal(library[0]?.about, "A book on how to plan ahead");
```

Comment:
For all tests checking for values we should use the propertyVal assert since that is more specific then assert.equal

https://www.chaijs.com/api/assert/#method_propertyval